content/guides/frameworks/laravel/development-setup.md

```diff
@@ -0,0 +1,342 @@
+---
+title: Laravel Development Setup with Docker Compose
+description: Set up a Laravel development environment using Docker Compose.
+weight: 20
+---
+
+## Development Environment Setup
+
+This guide demonstrates how to set up a development environment for a Laravel application using Docker and Docker Compose. This setup includes essential services like PHP-FPM, Nginx, and a database (using Postgres, with MySQL/MariaDB as alternatives), which enable you to develop in an isolated and consistent environment.
+
+> [!NOTE]
+> If you want to quickly test this setup without configuring everything manually, you can download the [Laravel Docker Examples](https://github.com/rw4lll/laravel-docker-examples) repository. It includes pre-configured examples for both development and production environments.
+
+### Project Structure
+
+Start by creating a project structure that includes both the Laravel application and Docker-related files:
+
+```plaintext
+my-laravel-app
+├── app/
+├── bootstrap/
+├── config/
+├── database/
+├── public/
+├── docker/
+│   ├── php-fpm
+│   │   └── Dockerfile
+│   │   └── entrypoint.sh
+│   ├── workspace
+│   │   └── Dockerfile
+│   └── nginx
+│       └── nginx.conf
+├── compose.yaml
+├── .dockerignore
+├── .env
+├── vendor/
+├── ...
+```
+
+This structure includes a typical Laravel app, with a `docker` directory for Docker-related files like `php-fpm` and `workspace` Dockerfiles, the `nginx.conf` config file, and the `compose.yaml` file to define the services.
```

Comment:
I think it's better to base this guide on the example app you have created. Right now you're approaching this in a very neutral way ("create a project structure") but the Dockerfile and `compose.yaml` examples provided in this guide make a lot of assumptions.

Please use the example app as the starting point for this guide. Also provide an overview of that application. It doesn't need to go into every detail, but do describe how it's designed and the components involved: the Laravel app itself, FastCGI, reverse proxy.